Libraries/CameraRoll/RCTAssetsLibraryRequestHandler.m

```diff
@@ -53,10 +53,20 @@ - (id)sendRequest:(NSURLRequest *)request
 
       ALAssetRepresentation *representation = [asset defaultRepresentation];
       NSInteger length = (NSInteger)representation.size;
+      
+      NSString *mimeType = nil;
+      switch (CGImageGetAlphaInfo([representation CGImageWithOptions:nil])) {
```

Comment:
Can you replace this switch with a call to the `RCTImageHasAlpha` function from RCTImageUtils.m? It seems to be the same logic.